organization/_posts/2021/2021-09-30-coordination.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "HSF Coordination Meeting #215, 30 September 2021"
+layout: plain_toc
+---
+
+Present/Contributing: Graeme Stewart, Serhan Mete, Nicole Skidmore, Attila Krasznahorkay, Eduardo Rodrigues, Krzysztof Genser, Efe Yazgan, Caterina Doglioni, Ben Morgan, Andrea Valassi, Josh McFayden, Michel Jouvin, Liz Sexton-Kennedy, Alexander Moreno, Andi Salzburger, Benedikt Hegner, Michael Hernandez Villanueva, Philippe Canal, Sudhir Malik, Teng Jay Khoo, Daniel Elvira
+
+Apologies: Kyle Knoepfel
+
+## News, general matters, announcements
+
+### HL-LHC Review Phase 2
+
+Editors are now putting the finishing touches to their input documents. They need to be finished *today* for delivery to the reviewers *tomorrow*.
+
+The finished versions are being put into [CERNBox](https://cernbox.cern.ch/index.php/s/QGfcgPkvVsC2p31).
+
+Graeme and Liz wrote a short introduction, that summarises the process.
+
+N.B. The review itself will take place 1-5 November, hybrid at CERN + Zoomlandia. Attendance is being clarified with the LHCC.
+
+### CHEP2022 -> CHEP2023
+
+As people will have seen, CHEP has been delayed for a year, to May 2023. The means that we would not have a CHEP-attached WLCG/HSF workshop in May next year.
+
+This does though give more scope for smaller (still hybrid) workshops next year; probably even more important to give the community an opportunity for presentations and discussions.
+
+We should gather some ideas of what suitable topics could be and start to discuss sounding things out / organising.
+
+Some ideas:
+
+- Mini-workshop on heterogeneous computing (suggested by Reco + Trigger WG)
+- Analysis WS (covering also analysis facilities and HL-LHC analysis needs)
+
+### HSF Letters
+
+Graeme's thoughts:
+
+Had not really considered as the HSF letters that would go to an individual before (this issue came up a few weeks ago, we decided not to give a letter of 'support' as this would imply endorsement of an individual, which is not something the HSF is well setup to do).
```

Comment:
```suggestion
Had not really considered the HSF letters that would go to an individual before (this issue came up a few weeks ago, we decided not to give a letter of 'support' as this would imply endorsement of an individual, which is not something the HSF is well set up to do).
```